client/src/components/modals/incorporate-login.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <v-dialog v-model="showModal" max-width="60%">
+    <v-card class="pa-4">
```

Comment:
Why use a v-card?

Reply:
Good catch, it is redundant here! Updated in next commit

Reply:
Sorry, to come back to this, the card was to follow the same design pattern of the other dialogs in this UI. Also, the contents was strictly the common component pulled in, which was just a container.

Reply:
Do you still need the padding?

Reply:
I feel yes, the common component doesn't have anything and it looks a little 'tight', you could say.

Reply:
Can you add `content-class` to the v-dialog to get your padding?
I think using a v-card is overkill :>

Reply:
I could, but every other modal follows this pattern, so for the sake of congruency i would prefer to follow the pattern already established. If you feel like this is a good exception though, i could for sure.

Reply:
I don't think it's a big deal either way, but I did notice you lost your "pa-4" class.

Reply:
😅 Thanks!